CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 All notable changes to this project will be documented in this file.
 
+### [18.0.1](https://github.com/terraform-aws-modules/terraform-aws-eks/compare/v18.0.0...v18.0.1) (2022-01-06)
```

Comment:
we can leave out any changes here, the release workflow will automatically update this

Reply:
My bad, needed to pull from upstream.

Reply:
ah, I see - no worries!